recipes/uvicorn/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "uvicorn" %}
+{% set version = "0.3.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8de03999a936d8704f07cc3b1d3a3edb6922a068b64d84b4f5e49604c8b70a11
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  skip: True  # [py2k]
+  entry_points:
+    - uvicorn = uvicorn.main:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - click
+    - h11
+    - websockets >=6.0
+    - uvloop  # [not win]
+    - httptools  # [not win]
+
+test:
+  imports:
+    - uvicorn
+  commands:
+    - uvicorn --help
+
+about:
+  home: https://github.com/encode/uvicorn
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.md'
```

Comment:
Could you please open a PR in this project to also include the ``LICENSE.md`` in the next release?

Reply:
will do! Thanks

Reply:
done: https://github.com/encode/uvicorn/pull/230